apps/api/src/wallets/wallets.controller.ts

```diff
@@ -20,7 +21,7 @@ export class WalletsController {
     return await this.walletsService.createWallet(req.user, dto);
   }
 
-  @Post()
+  @Put()
```

Comment:
Chỗ này không nên dùng Put

Put nên dùng khi **thay thế hoàn toàn** cái cũ (ví dụ upload hình ảnh, file, file nhạc, ...)

Khi chỉ **thay đổi một phần, giữ lại những thứ khác**, mình nên dùng Post hoặc Patch. Do trong project này dùng Patch cho update, ở nên dùng Patch để cho đồng nhất

Reply:
@MinhKhang0715 GJ! chỉ cần sửa chút xíu thôi

Reply:
Xong rồi nha